doc/FAQ.org

```diff
@@ -282,6 +305,28 @@ To restore the sentence delimiter to two spaces, add the following code to the
 (setq sentence-end-double-space t)
 #+end_src
 
+** Change the initial mode of the =*scratch*= buffer?
+To change the initial mode of the =*scratch*= buffer, add the following code to
+the =dotspacemacs/user-config= function of your =.spacemacs=:
```

Comment:
This tip doesn't actually work. By the time `user-config` is run, the scratch buffer is already created.